core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -335,6 +335,9 @@ public enum Property {
           + " was changed and it now can accept multiple class names. The metrics spi was introduced in 2.1.3,"
           + " the deprecated factory is org.apache.accumulo.core.metrics.MeterRegistryFactory.",
       "2.1.0"),
+  GENERAL_SERVER_ITERATOR_OPTIONS_COMPRESSION_ALGO("general.server.iter.opts.compression", "none",
```

Comment:
probably need to add this to the fixed set of properties as we likely don't want this changed at runtime. Wondering if we need to add it to the instance properties to make sure it's the same across the cluster.

Reply:
that's right, I forgot that the algo was sent in the options map.